health/micro-ui/web/micro-ui-internals/packages/modules/health-hrms/src/pages/employee/createEmployee.js

```
@@ -73,6 +73,8 @@ const CreateEmployee = ({ editUser = false }) => {
   };
 
   const onFormValueChange = (setValue = true, formData, formState, reset, setError, clearErrors) => {
+    debugger
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)